### PR TITLE
[FIF-115] Build configurations for Angular

### PR DESCRIPTION
**Pull request by stephenfuqua**
I converted the Angular project to use `yarn` because that is our standard, and added the necessary configurations options for running tests and lint with appropriate reporting in TeamCity.

### CODE REVIEW

**Comment by stephenfuqua** [comment]
Requesting @kurtjohnson-HS review of Kotlin and @dougloyo review of tweaks to the Angular project configuration

File: .teamcity/Angular/buildTypes/BranchUIBuild.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        powerShell {
+            name = "Package"
+            id = "BranchUIBuild_Package"
+            workingDir = "%project.directory%/eng"
+            formatStderrAsError = true
+            scriptMode = script {
+                content = """
+                    .\build-package.ps1 -BuildCounter %build.counter%
+                """.trimIndent()
+            }
```
Comment (stephenfuqua) [inline .teamcity/Angular/buildTypes/BranchUIBuild.kt:31]: Hm, I forgot to create a build package script. Better fix that now.

Comment (stephenfuqua) [reply .teamcity/Angular/buildTypes/BranchUIBuild.kt:31]: done.

**Comment by kurtjohnson-HS** [comment]
This failed on INETFIBUILD1. Can we skip building on that target altogether?

**Comment by stephenfuqua** [comment]
> This failed on INETFIBUILD1. Can we skip building on that target altogether?

I believe the [failed build](https://intedfitools1.msdf.org/buildConfiguration/EdFi_AnalyticsTeam_FixItFriday_PullRequestAPIBuild/74962?buildTab=log&focusLine=2&linesState=205) triggered because of changes in the Kotlin itself, rather than triggering due to the Angular project.

The problem in that case is that server `intedfibuild1` needs a Node version update. I've just updated that server, which [has resolved](https://intedfitools1.msdf.org/buildConfiguration/EdFi_AnalyticsTeam_FixItFriday_PullRequestAPIBuild/75418?buildTab=log&expandAll=true&focusLine=115) that particular issue.

**Pull request merged**